openlibrary/components/MergeUI/MergeTable.vue

```diff
@@ -153,6 +153,16 @@ export default {
             );
         },
 
+        async find_unretrieved_editions() {
```

Comment:
This is a `computed`, so behaves like a property more than a function actually. So we don't want to name it as a verb, because that makes it seem like a function, when it's more of a property with a backing getter.

And we're not retrieving editions, we're retrieving the works, so `unmergeable_works` seems like a better name.

```suggestion
        async unmergeable_works() {
```

Reply:
Actually, I think it's clearer if we put this logic inside the `merge` data structure. So we can have a field of `unmergeable_works` inside the return of the `async merge` function.